presto-main/src/test/java/io/prestosql/server/TestBasicQueryInfo.java

```diff
@@ -63,6 +63,7 @@ public void testConstructor()
                                 Duration.valueOf("35m"),
                                 Duration.valueOf("44m"),
                                 Duration.valueOf("9m"),
+                                Duration.valueOf("10s"),
```

Comment:
Maybe pick a different, unique, value.

Reply:
Any suggestion? What is wrong here. It would be nice that all the values here would be unique and follow some pattern so it could be ease to validate the assertion. However, because all the changes around this code the pattern is no longer visible. So I tried to use an unique value.

Reply:
`99`?